src/components/InGameMenu.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import Card from "./Card";
+import GameButton from "./GameButton";
+import { colors, HeadingL } from "../styles/theme";
+
+const Overlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+`;
+
+const MenuButton = styled(GameButton)`
+  justify-content: center;
+`;
+
+const InGameMenu = ({ isOpen, onClose, restart, quit }) => {
+  if (!isOpen) return null;
+  return (
+    <Overlay>
+      <Card>
+        <HeadingL $textColor={colors.white}>Pause</HeadingL>
+        <MenuButton label="Continue game" onClick={onClose} />
+        <MenuButton label="Restart" onClick={restart} />{" "}
```

Comment:
```suggestion
        <MenuButton label="Restart" onClick={restart} />
```

 c'est nécéssaire le (" "} ?